leakcanary-android/src/main/java/com/squareup/leakcanary/internal/DisplayLeakActivity.java

```diff
@@ -364,9 +376,29 @@ public void onItemClick(AdapterView<?> parent, View view, int position, long id)
         });
       }
       actionButton.setVisibility(leaks.size() == 0 ? GONE : VISIBLE);
+      shareButton.setVisibility(GONE);
     }
   }
 
+  private void shareLeakToStackOverflow() {
+    AnalyzedHeap visibleLeak = getVisibleLeak();
+    final String leakInfo = LeakCanary.INSTANCE.leakInfo(this, visibleLeak.heapDump, visibleLeak.result, false);
```

Comment:
I've set the `detailed` parameter to `false` right now as when it is `true` I saw that the leak info can exceed the 30000 character limit that Stack Overflow has on question bodies. Open to other suggestions to handle this better.

Reply:
You made the right call (this will be going away anyway)